GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg

```diff
@@ -264,7 +264,9 @@
 	@breakingForce = 2000
 	@breakingTorque = 2000
 	%maxTemp = 400
-	%skinMaxTemp = 900 // Mach 3
+	%skinMaxTemp = 600
```

Comment:
Have these numbers been tested in game to be equivalent to the old skin temperature?

Reply:
they've been tested in-game to the extent described in the commit message:

- allow spending at least a bit of time (tested 3 minutes) at mach 3 without melting down
- enough to stay at ~mach 2.2 indefinitely